<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mirador Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Port : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="portListen">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Password :</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QLineEdit" name="lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>124</width>
            <height>0</height>
           </size>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="listenBut">
          <property name="text">
           <string>Listen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>listenBut</tabstop>
  <tabstop>listenBut</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>portListen</tabstop>
  <tabstop>portListen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
